apps/public-docsite/src/SiteDefinition/SiteDefinition.pages/Controls/web.tsx

```diff
@@ -62,6 +62,7 @@ export const categories: { Other?: ICategory; [name: string]: ICategory } = {
         CustomFooter: { title: 'Custom Footer' },
         CustomGroupHeaders: { title: 'Custom Group Headers' },
         Advanced: { title: 'Variable Row Heights', url: 'variablerowheights' },
+        KeyboardDragDrop: { title: 'Keyboard Drag & Drop', url: 'keyboarddraganddrop' },
```

Comment:
It might be better if you can think of a concise title that indicates this is specifically for drag/drop/resize of columns (not items)

Reply:
Changed it to "Keyboard Column Reorder & Resize"